R/format_commit_shas.R

```diff
@@ -9,8 +9,9 @@
 
 get_commits_df <- function(issue_number, owner = get_organization(), repo = get_current_repo()) {
   init_qc_commit <- get_init_qc_commit(owner, repo, issue_number)
+  branch <- get_branch_from_metadata(owner, repo, issue_number)
 
-  all_commits <- gert::git_log()
+  all_commits <- gert::git_log(ref = paste0("origin/", branch))
```

Comment:
are we guaranteeing/expecting the remote to be the origin? IMO this should be an option we set with a default of origin. Something like ghqcapp.remote_name, so this would become ~ getOption("ghqcapp.remote_name", "origin")

Reply:
Good call - yes, it already is an option!

Kun said that this fix worked for him, so I'll tidy it up. I'll also try to think of a way to not make this branch-dependent, but I'm not confident it's possible